hyrax/db/schema.rb

```diff
@@ -548,8 +548,10 @@
     t.string "unlock_token"
     t.datetime "locked_at"
     t.string "remember_token"
+    t.string "user_identifier"
```

Comment:
this probably should be a uuid data type rather than a string? see https://guides.rubyonrails.org/active_record_postgresql.html#uuid

Reply:
Unless of course this is not a uuid, but is in fact a canonical id "urn:USER_IDENTIFIER.dpfc.nims.go.jp:xxxx" - in which case `string` is the right datatype :-)